### ISSUE TITLE
Can't split URI

### ISSUE THREAD

**Issue by fils** [costezki/rdf2gremlin#16]
So I fired up Janus graph  and attempted to load some RDF data.  However, I get an error.  

Any guidance or help would be appreciated

The error from the run
```
 (gremlin)  fils@xps  ~/src/Projects/ResearchItems/JanusGraph  python loader.py
2020-05-14 12:04:40,528 [INFO] Successfully connected to the graph server
Traceback (most recent call last):
  File "loader.py", line 16, in <module>
    load_rdf2g(g, rdf_graph)
  File "/home/fils/src/python/venvs/gremlin/lib/python3.6/site-packages/rdf2g/update.py", line 30, in load_rdf2g
    subj_node = create_node(g, s, rdf_graph)
  File "/home/fils/src/python/venvs/gremlin/lib/python3.6/site-packages/rdf2g/update.py", line 69, in create_node
    label = rdf_graph.qname(rdf_term) if rdf_graph.qname(rdf_term) else str(rdf_term)
  File "/home/fils/src/python/venvs/gremlin/lib/python3.6/site-packages/rdflib/graph.py", line 904, in qname
    return self.namespace_manager.qname(uri)
  File "/home/fils/src/python/venvs/gremlin/lib/python3.6/site-packages/rdflib/namespace.py", line 294, in qname
    prefix, namespace, name = self.compute_qname(uri)
  File "/home/fils/src/python/venvs/gremlin/lib/python3.6/site-packages/rdflib/namespace.py", line 330, in compute_qname
    namespace, name = split_uri(uri)
  File "/home/fils/src/python/venvs/gremlin/lib/python3.6/site-packages/rdflib/namespace.py", line 502, in split_uri
    raise Exception("Can't split '%s'" % uri)
Exception: Can't split 'N80115c7e6093451a80be8ce53a2abbb4'
```

from janusgraph
```
...
389771 [gremlin-server-exec-7] WARN  org.janusgraph.graphdb.transaction.StandardJanusGraphTx  - Query requires iterating over all vertices [(iri = http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0)]. For better performance, use indexes
389789 [gremlin-server-exec-1] WARN  org.janusgraph.graphdb.transaction.StandardJanusGraphTx  - Query requires iterating over all vertices [(iri = http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0)]. For better performance, use indexes
389796 [gremlin-server-exec-2] WARN  org.janusgraph.graphdb.transaction.StandardJanusGraphTx  - Query requires iterating over all vertices [(iri = http://sample.igsn.org/soilarchive/CDS-NSW)]. For better performance, use indexes
389812 [gremlin-server-exec-4] WARN  org.janusgraph.graphdb.transaction.StandardJanusGraphTx  - Query requires iterating over all vertices [(iri = http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60)]. For better performance, use indexes
389829 [gremlin-server-exec-6] WARN  org.janusgraph.graphdb.transaction.StandardJanusGraphTx  - Query requires iterating over all vertices [(iri = http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60)]. For better performance, use indexes
389835 [gremlin-server-exec-7] WARN  org.janusgraph.graphdb.transaction.StandardJanusGraphTx  - Query requires iterating over all vertices [(iri = http://samples.earth/doc/bqs2dmqu6s73o70jds60)]. For better performance, use indexes
389860 [gremlin-server-exec-2] WARN  org.janusgraph.graphdb.transaction.StandardJanusGraphTx  - Query requires iterating over all vertices [(iri = N80115c7e6093451a80be8ce53a2abbb4)]. For better performance, use indexes

```

the code
```python
from rdf2g import setup_graph
import rdflib
import pathlib
from rdf2g import load_rdf2g


DEFAULT_LOCAL_CONNECTION_STRING = "ws://localhost:8182/gremlin"
g = setup_graph(DEFAULT_LOCAL_CONNECTION_STRING)


OUTPUT_FILE_LAM_PROPERTIES = pathlib.Path("./test.nq").resolve()

rdf_graph = rdflib.Graph()
rdf_graph.parse(str(OUTPUT_FILE_LAM_PROPERTIES), format="nquads")

load_rdf2g(g, rdf_graph)

```

the test nq
```
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://purl.org/dc/terms/title> "ANZ soil sample bqs2dj2u6s73o70jcpr0" .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://schema.org/additionalType> <http://pid.geoscience.gov.au/def/voc/igsn-codelists/PhysicalSample> .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://schema.org/additionalType> <http://pid.geoscience.gov.au/def/voc/igsn-codelists/soil> .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://schema.org/creator> <http://sample.igsn.org/soilarchive/CDS-NSW> .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://schema.org/dateCreated> "1970-07-21T05:03:19-05:00"^^<http://www.w3.org/2001/XMLSchema#date> .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://schema.org/description> "assumenda eum velit ea molestias quia explicabo! possimus cum quam ea qui saepe quaerat. assumenda autem qui omnis in. minima impedit rerum dolorum ullam eos nemo. quidem laudantium quo. ipsam et saepe rerum voluptas perferendis. iste unde maiores magni occaecati. cumque magnam suscipit. facere ipsa iusto magni doloremque iste. qui qui dolorem ea numquam quia alias voluptatem." .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://schema.org/spatialCoverage> _:bbqs2gpqu6s78kvjb1ee0 .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://schema.org/title> "ANZ soil sample" .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://schema.org/url> <http://samples.earth/id/do/bqs2dj2u6s73o70jcpr0> .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2002/07/owl#Thing> .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/sosa/Sample> .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://www.w3.org/2000/01/rdf-schema#label> "ANZ soil sample" .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://www.w3.org/ns/dcat#landingPage> <http://samples.earth/doc/bqs2dj2u6s73o70jcpr0> .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://www.w3.org/ns/sosa/isResultOf> _:bbqs2gpqu6s78kvjb1eeg .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://www.w3.org/ns/sosa/isSampleOf> <http://www.anzsoil.org/data/csiro-natsoil/anzsoilml201/soil/soil_199.CAN.C410> .
<http://sample.igsn.org/soilarchive/bqs2dj2u6s73o70jcpr0> <http://www.w3.org/ns/sosa/isSampleOf> <http://www.anzsoil.org/data/csiro-natsoil/anzsoilml201/soilhorizon/soil_horizon_199.CAN.C410.1.2> .
_:bbqs2gpqu6s78kvjb1ee0 <http://schema.org/geo> _:bbqs2gpqu6s78kvjb1ef0 .
_:bbqs2gpqu6s78kvjb1ee0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Place> .
_:bbqs2gpqu6s78kvjb1ef0 <http://schema.org/latitude> "23"^^<http://www.w3.org/2001/XMLSchema#integer> .
_:bbqs2gpqu6s78kvjb1ef0 <http://schema.org/longitude> "40"^^<http://www.w3.org/2001/XMLSchema#integer> .
_:bbqs2gpqu6s78kvjb1ef0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/GeoCoordinates> .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://purl.org/dc/terms/title> "ANZ soil sample bqs2dniu6s73o70je5g0" .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://schema.org/additionalType> <http://pid.geoscience.gov.au/def/voc/igsn-codelists/PhysicalSample> .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://schema.org/additionalType> <http://pid.geoscience.gov.au/def/voc/igsn-codelists/soil> .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://schema.org/creator> <http://sample.igsn.org/soilarchive/CDS-NSW> .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://schema.org/dateCreated> "1985-07-23T08:58:03-05:00"^^<http://www.w3.org/2001/XMLSchema#date> .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://schema.org/description> "atque id perspiciatis sapiente voluptatem velit ullam velit earum." .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://schema.org/spatialCoverage> _:bbqs2gpqu6s78kvjb1e80 .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://schema.org/title> "ANZ soil sample" .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://schema.org/url> <http://samples.earth/id/do/bqs2dniu6s73o70je5g0> .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2002/07/owl#Thing> .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/sosa/Sample> .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://www.w3.org/2000/01/rdf-schema#label> "ANZ soil sample" .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://www.w3.org/ns/dcat#landingPage> <http://samples.earth/doc/bqs2dniu6s73o70je5g0> .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://www.w3.org/ns/sosa/isResultOf> _:bbqs2gpqu6s78kvjb1e8g .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://www.w3.org/ns/sosa/isSampleOf> <http://www.anzsoil.org/data/csiro-natsoil/anzsoilml201/soil/soil_199.CAN.C410> .
<http://sample.igsn.org/soilarchive/bqs2dniu6s73o70je5g0> <http://www.w3.org/ns/sosa/isSampleOf> <http://www.anzsoil.org/data/csiro-natsoil/anzsoilml201/soilhorizon/soil_horizon_199.CAN.C410.1.2> .
_:bbqs2gpqu6s78kvjb1e80 <http://schema.org/geo> _:bbqs2gpqu6s78kvjb1e90 .
_:bbqs2gpqu6s78kvjb1e80 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Place> .
_:bbqs2gpqu6s78kvjb1e90 <http://schema.org/latitude> "83"^^<http://www.w3.org/2001/XMLSchema#integer> .
_:bbqs2gpqu6s78kvjb1e90 <http://schema.org/longitude> "-178"^^<http://www.w3.org/2001/XMLSchema#integer> .
_:bbqs2gpqu6s78kvjb1e90 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/GeoCoordinates> .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://purl.org/dc/terms/title> "ANZ soil sample bqs2dhqu6s73o70jcdg0" .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://schema.org/additionalType> <http://pid.geoscience.gov.au/def/voc/igsn-codelists/PhysicalSample> .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://schema.org/additionalType> <http://pid.geoscience.gov.au/def/voc/igsn-codelists/soil> .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://schema.org/creator> <http://sample.igsn.org/soilarchive/CDS-NSW> .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://schema.org/dateCreated> "1997-08-17T07:18:08-05:00"^^<http://www.w3.org/2001/XMLSchema#date> .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://schema.org/description> "voluptatibus et totam voluptate voluptatem. et voluptas quidem quo. voluptatum aliquam aliquid ad. hic accusamus dolor. repellat vitae rerum rem et est labore. praesentium dolorum numquam voluptas. minima expedita quia quia ut rerum corporis amet rem tempore. facere quisquam quo in fuga sunt eos quam." .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://schema.org/spatialCoverage> _:bbqs2gpqu6s78kvjb1elg .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://schema.org/title> "ANZ soil sample" .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://schema.org/url> <http://samples.earth/id/do/bqs2dhqu6s73o70jcdg0> .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2002/07/owl#Thing> .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/sosa/Sample> .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://www.w3.org/2000/01/rdf-schema#label> "ANZ soil sample" .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://www.w3.org/ns/dcat#landingPage> <http://samples.earth/doc/bqs2dhqu6s73o70jcdg0> .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://www.w3.org/ns/sosa/isResultOf> _:bbqs2gpqu6s78kvjb1em0 .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://www.w3.org/ns/sosa/isSampleOf> <http://www.anzsoil.org/data/csiro-natsoil/anzsoilml201/soil/soil_199.CAN.C410> .
<http://sample.igsn.org/soilarchive/bqs2dhqu6s73o70jcdg0> <http://www.w3.org/ns/sosa/isSampleOf> <http://www.anzsoil.org/data/csiro-natsoil/anzsoilml201/soilhorizon/soil_horizon_199.CAN.C410.1.2> .
_:bbqs2gpqu6s78kvjb1elg <http://schema.org/geo> _:bbqs2gpqu6s78kvjb1emg .
_:bbqs2gpqu6s78kvjb1elg <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Place> .
_:bbqs2gpqu6s78kvjb1emg <http://schema.org/latitude> "-13"^^<http://www.w3.org/2001/XMLSchema#integer> .
_:bbqs2gpqu6s78kvjb1emg <http://schema.org/longitude> "-137"^^<http://www.w3.org/2001/XMLSchema#integer> .
_:bbqs2gpqu6s78kvjb1emg <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/GeoCoordinates> .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://purl.org/dc/terms/title> "ANZ soil sample bqs2dmqu6s73o70jds60" .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://schema.org/additionalType> <http://pid.geoscience.gov.au/def/voc/igsn-codelists/PhysicalSample> .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://schema.org/additionalType> <http://pid.geoscience.gov.au/def/voc/igsn-codelists/soil> .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://schema.org/creator> <http://sample.igsn.org/soilarchive/CDS-NSW> .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://schema.org/dateCreated> "2003-03-18T02:42:35-06:00"^^<http://www.w3.org/2001/XMLSchema#date> .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://schema.org/description> "hic atque tempore sint animi." .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://schema.org/spatialCoverage> _:bbqs2gpqu6s78kvjb1efg .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://schema.org/title> "ANZ soil sample" .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://schema.org/url> <http://samples.earth/id/do/bqs2dmqu6s73o70jds60> .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2002/07/owl#Thing> .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/sosa/Sample> .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://www.w3.org/2000/01/rdf-schema#label> "ANZ soil sample" .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://www.w3.org/ns/dcat#landingPage> <http://samples.earth/doc/bqs2dmqu6s73o70jds60> .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://www.w3.org/ns/sosa/isResultOf> _:bbqs2gpqu6s78kvjb1eg0 .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://www.w3.org/ns/sosa/isSampleOf> <http://www.anzsoil.org/data/csiro-natsoil/anzsoilml201/soil/soil_199.CAN.C410> .
<http://sample.igsn.org/soilarchive/bqs2dmqu6s73o70jds60> <http://www.w3.org/ns/sosa/isSampleOf> <http://www.anzsoil.org/data/csiro-natsoil/anzsoilml201/soilhorizon/soil_horizon_199.CAN.C410.1.2> .
_:bbqs2gpqu6s78kvjb1efg <http://schema.org/geo> _:bbqs2gpqu6s78kvjb1egg .
_:bbqs2gpqu6s78kvjb1efg <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Place> .
_:bbqs2gpqu6s78kvjb1egg <http://schema.org/latitude> "26"^^<http://www.w3.org/2001/XMLSchema#integer> .
_:bbqs2gpqu6s78kvjb1egg <http://schema.org/longitude> "-33"^^<http://www.w3.org/2001/XMLSchema#integer> .
_:bbqs2gpqu6s78kvjb1egg <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/GeoCoordinates> .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://purl.org/dc/terms/title> "ANZ soil sample bqs2dsqu6s73o70jfjb0" .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://schema.org/additionalType> <http://pid.geoscience.gov.au/def/voc/igsn-codelists/PhysicalSample> .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://schema.org/additionalType> <http://pid.geoscience.gov.au/def/voc/igsn-codelists/soil> .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://schema.org/creator> <http://sample.igsn.org/soilarchive/CDS-NSW> .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://schema.org/dateCreated> "2005-08-02T09:18:28-05:00"^^<http://www.w3.org/2001/XMLSchema#date> .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://schema.org/description> "nesciunt et iste nisi atque et. consequatur enim dicta neque. vel error quos voluptatem in sit facere quo tenetur deserunt. voluptas tenetur architecto minus esse quam. sed dolorem nobis magnam. est culpa non placeat." .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://schema.org/spatialCoverage> _:bbqs2gpqu6s78kvjb1e9g .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://schema.org/title> "ANZ soil sample" .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://schema.org/url> <http://samples.earth/id/do/bqs2dsqu6s73o70jfjb0> .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2002/07/owl#Thing> .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/sosa/Sample> .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://www.w3.org/2000/01/rdf-schema#label> "ANZ soil sample" .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://www.w3.org/ns/dcat#landingPage> <http://samples.earth/doc/bqs2dsqu6s73o70jfjb0> .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://www.w3.org/ns/sosa/isResultOf> _:bbqs2gpqu6s78kvjb1ea0 .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://www.w3.org/ns/sosa/isSampleOf> <http://www.anzsoil.org/data/csiro-natsoil/anzsoilml201/soil/soil_199.CAN.C410> .
<http://sample.igsn.org/soilarchive/bqs2dsqu6s73o70jfjb0> <http://www.w3.org/ns/sosa/isSampleOf> <http://www.anzsoil.org/data/csiro-natsoil/anzsoilml201/soilhorizon/soil_horizon_199.CAN.C410.1.2> .
```

**Comment by costezki** [comment]
Apparently the data you try to load contains an invalid URI "N80115c7e6093451a80be8ce53a2abbb4". 
I would check first where it is a valid RDF dataset.

**Comment by fils** [comment]
I thought that too..   but that string is not in my RDF graph anywhere.  

Also  the string it claims it can not split  _changes every run.._

```
python loader.py
2020-05-15 09:11:41,825 [INFO] Successfully connected to the graph server
Traceback (most recent call last):
 [...]
Exception: Can't split 'N44f091eedab142e7890ea55110d1f77f'

python loader.py
2020-05-15 09:12:42,307 [INFO] Successfully connected to the graph server
Traceback (most recent call last):
[...]
Exception: Can't split 'N08376a48697c48a5bfe6771787502ecb'
```

each time I run the code it returns a different value.....

**Comment by costezki** [comment]
Blank nodes were not foreseen, only QNames; and therefore the code for node creation was failing. 
This is fixed now, in version 0.1.37.
